src/desktop/apps/personalize/routes.js

```diff
@@ -1,25 +1,10 @@
 import * as _ from 'underscore'
-import { renderLayout } from '@artsy/stitch'
+import { renderLayout as _renderLayout } from '@artsy/stitch'
 import { App } from 'desktop/apps/personalize/components/App'
 
-export const index = (req, res, next) => {
-  if (res.locals.sd.ONBOARDING_TEST !== 'experiment') {
-    req.user.fetch({
-      success: (model, response, options) => {
-        res.locals.sd.CURRENT_USER = _.extend(
-          {},
-          response,
-          res.locals.sd.CURRENT_USER
-        )
-        res.render('template')
-      },
-    })
-  } else {
-    newOnboarding(req, res, next)
-  }
-}
+let renderLayout = _renderLayout
```

Comment:
What's this for?

Reply:
@kanaabe in the test set up `renderLayout` returns an **undefined error** when it get's passed to the `rewire` function. I saw this renaming wizardry in the [auctions app](https://github.com/artsy/force/blob/master/src/desktop/apps/auction/routes.js#L18-L22) and it seems to make `renderLayout` available.